jena-arq/src/test/java/org/apache/jena/sparql/api/TestCancelQueryAFS.java

```diff
@@ -0,0 +1,155 @@
+package org.apache.jena.sparql.api;
+
+/*
+ *  Copyright (c) Telicent Ltd.
```

Comment:
License header likely needs adjustment (this is the only file with a Telicent header)